test/e2e/network/ingress.go

```diff
@@ -947,50 +947,53 @@ var _ = SIGDescribe("Ingress API", func() {
 		Testname: Ingress API
 		Description:
 		The networking.k8s.io API group MUST exist in the /apis discovery document.
-		The networking.k8s.io/v1beta1 API group/version MUST exist in the /apis/networking.k8s.io discovery document.
-		The ingresses resources MUST exist in the /apis/networking.k8s.io/v1beta1 discovery document.
+		The networking.k8s.io/v1 API group/version MUST exist in the /apis/networking.k8s.io discovery document.
+		The ingresses resources MUST exist in the /apis/networking.k8s.io/v1 discovery document.
 		The ingresses resource must support create, get, list, watch, update, patch, delete, and deletecollection.
 		The ingresses/status resource must support update and patch
-
 	*/
 
-	ginkgo.It("should support creating Ingress API operations", func() {
+	framework.ConformanceIt("should support creating Ingress API operations", func() {
 		// Setup
 		ns := f.Namespace.Name
-		ingVersion := "v1beta1"
-		ingClient := f.ClientSet.NetworkingV1beta1().Ingresses(ns)
-
-		prefixPathType := networkingv1beta1.PathTypePrefix
+		ingVersion := "v1"
+		ingClient := f.ClientSet.NetworkingV1().Ingresses(ns)
+
+		prefixPathType := networkingv1.PathTypePrefix
+		serviceBackend := &networkingv1.IngressServiceBackend{
+			Name: "defaultbackend",
+			Port: networkingv1.ServiceBackendPort{
+				Name:   "",
+				Number: 80,
```

Comment:
to be equivalent, should be:
```suggestion
				Number: 8080,
```